### PR TITLE
Lpfg 636 add course confirmation

### PR DESCRIPTION
**Pull request by mick-jones**
- Included 'course added' panel on course overview page

### CODE REVIEW

File: views/page/course.html [hunk 1]
```diff
@@ -17,6 +17,13 @@
 {% block content %}
 <div class="govuk-grid-row">
     <div class="govuk-grid-column-two-thirds">
+        {% if courseAddedSuccessMessage %}
+            <div class="panel govuk-panel govuk-panel--confirmation">
+                <div class="govuk-panel__body custom-panel">
+                    <p class="govuk-body"><strong>{{ course.title }}</strong>{{courseAddedSuccessMessage}}</p>
+                </div>
+            </div>
+        {% endif %}
```
[the remark this answers is not in the document]
Comment (m24murray) [reply views/page/course.html]: @superafroman - Had a chat with Joel on this one, so the `panel` component on the new gov kit is for completing "transactions" in conjunction with a confirmation page (like how we did booking a classroom course): https://design-system.service.gov.uk/patterns/confirmation-pages/

Currently gov doesn't seem to have a native component for these smaller banners, so we need to modify and roll our own.

_tl;dr: It's too big and ugly and doesn't suit the pattern we are implementing here. Joel says is fine._

**Comment by m24murray** [comment]
Closing and replacing with https://github.com/Civil-Service-Human-Resources/lpg-management/pull/31
